src/webview/helm-chart/app/cardItem.tsx

```diff
@@ -117,98 +118,162 @@ export class CardItem extends React.Component<DevFileProps, {
-                                    className={
-                                        selectedVersion.version === chart.version ? this.props.cardItemStyle.starterProjectSelected : this.props.cardItemStyle.project
-                                    }
-                                >
-                                    {chart.version}
-                                </div>
-                            ))}
+                    <div style={{ width: '50%', minHeight: '5rem', maxHeight: '10rem' }}>
+                        {selectedVersion.description && <div className={this.props.cardItemStyle.helmCardDetails}>
+                            <Typography variant='body2' className={this.props.cardItemStyle.detailedDescription}>
+                                <p className={this.props.cardItemStyle.helmCardDetailItem}>Description:</p>
+                                <p className={this.props.cardItemStyle.helmCardDetailItemValue}>
+                                    {selectedVersion.description}
+                                </p>
+                            </Typography>
+                        </div>
+                        }
+                        <div className={this.props.cardItemStyle.helmCardDetails} style={{ height: '4rem' }}>
+                            <Typography variant='body2' className={this.props.cardItemStyle.detailedDescription}>
+                                <p className={this.props.cardItemStyle.helmCardDetailItem}>Product Version:</p>
+                                <p className={this.props.cardItemStyle.helmCardDetailItemValue}>{selectedVersion.appVersion || selectedVersion.version}</p>
+                            </Typography>
+                        </div>
+                        {
+                            selectedVersion.annotations['charts.openshift.io/supportURL'] &&
+                            <div className={this.props.cardItemStyle.helmCardDetails} style={{ height: '4rem' }}>
+                                <Typography variant='body2' className={this.props.cardItemStyle.detailedDescription}>
+                                    <p className={this.props.cardItemStyle.helmCardDetailItem}>Home Page:</p>
+                                    <a className={this.props.cardItemStyle.helmCardDetailItemValue} href={selectedVersion.annotations['charts.openshift.io/supportURL']}>{selectedVersion.annotations['charts.openshift.io/supportURL']}</a>
+                                </Typography>
+                            </div>
+                        }
+                        <div className={this.props.cardItemStyle.helmCardDetails} style={{ height: '4rem' }}>
+                            <Typography variant='body2' className={this.props.cardItemStyle.detailedDescription}>
+                                <p className={this.props.cardItemStyle.helmCardDetailItem}>Repository:</p>
+                                <p className={this.props.cardItemStyle.helmCardDetailItemValue}>OpenShift Helm Charts</p>
```

Comment:
@msivasubramaniaan If this value is hard coded and true for all Helm Charts, we do not need to show this. Can we get a URL here from response ?

Reply:
@msivasubramaniaan If this value is hard coded and true for all Helm Charts, we do not need to show this. Can we get a URL here from response ?

Reply:
Currently it will not in the response. so removed it